npc/other/Global_Functions.txt

```diff
@@ -659,3 +660,97 @@ function	script	F_getpositionname	{
 			return "Unknown";
 	}
 }
+
+//////////////////////////////////////////////////////////////////////////////////
+// Platinum Skills Global Function: F_GetPlatinumSkills [mazvi] 
+//////////////////////////////////////////////////////////////////////////////////
+function	script	F_GetPlatinumSkills	{
+	skill "NV_FIRSTAID",1,SKILL_PERM;
```

Comment:
Add a check for summoner class. They shouldn’t get NV_FIRSTAID.

Reply:
check using method novice class check or summoner class check?

Reply:
doram check has been added

Reply:
it should be **break;** just in case ? **end;** wont hit the **return;**

Reply:
I double-checked and doram can acquire NV_FIRSTAID in lasagna town - that's why I removed my previous comment about it but maybe lead @secretdataz to the wrong direction.
Please remove this check.

Reply:
doram check has been removed